android/src/androidTest/java/io/rownd/android/AuthInstrumentedTest.kt

```diff
@@ -261,4 +262,96 @@ class AuthInstrumentedTest {
         }
     }
 
+    @Test
+    fun detect_expired_access_token() = runTest {
```

Comment:
For all of these tests, I think we should assert that `state.auth.isAccessTokenValid` is `true/false`.

Reply:
Added